sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj

```diff
@@ -9,10 +9,8 @@
 		<![CDATA[
 			Documentation can be found at https://go.microsoft.com/fwlink/p/?linkid=862452.
 			
-			Support for new App Services managed identity API
-			Key vault client updates
-			Enabling CRL checks
-			Bug fixes
+			Support for managed identies on Service Fabric
+			Improvements and bug fixes for token providers
```

Comment:
Should the package version be incremented?

Reply:
It was incremented in an early PR to 1.6. That PR missed updating the AssemblyInfo.cs which is in this PR.